### DISCUSSION TITLE
Status of WIndows

### DISCUSSION THREAD

**Discussion by jlorezz** [zed-industries/zed#7750] [Feature Requests]
Hey! What's the eta of windows support? I really like the idea behind Zed and would love to give it a try ;)

**Comment by SomeoneToIgnore** [comment]
Hey, there's no ETA right now — Zed team is concentrated on Linux target as the first platform to support, Windows should follow later at some point.

There are people attempting to improve Windows support, consider https://github.com/zed-industries/zed/discussions/7944 and https://github.com/zed-industries/zed/pull/7827 and those contributions are welcome, but seems that those are in very early stages, as the code does not even compile and there's no CI to ensure it stays compiling after all fixes submitted.
